docs/ja/front/component/ma-form.md

```
@@ -1,93 +1,313 @@
-| `getOptions()`      | `ma-form` 設定を取得   | -                 | `MaFormOptions` |
-| `setItems()`        | フォーム項目を設定             | `(MaFormItem[])`  | -               |
-| `getItems()`        | フォーム項目を取得             | -                 | `MaFormItem[]`  |
-| `appendItem()`      | フォーム項目を追加             | `(MaFormItem)`    | -               |
-| `removeItem()`      | フォーム項目を削除             | `(prop: string)`  | -              |
-| `getItemByProp()`   | `prop`でフォーム項目を取得      | `(prop: string)`  | `MaFormItem`    |
-| `isMobileState()`   | モバイルモードかどうかを確認        | -                 | `boolean`    |
-| `getElFormRef()`    | `el-form` Ref を取得  | -                 | `El-Form`       |
\ No newline at end of file
+| `cols`        | グリッドレイアウト設定、`layout` が `flex` の場合に有効、レスポンシブ設定をサポート                                                                                                                                                  | `ColsConfiguration`                    | -       | 1.0.0  |
+| `itemProps`   | Element Plus `el-form-item` のネイティブプロパティ                                                                                                                                                                            | `ElFormItemProps`                    | `{}`       | 1.0.0  |
+| `itemSlots`   | Element Plus `el-form-item` のネイティブスロット設定                                                                                                                                                                            | `Record<string, (...args: any[]) => VNode \| VNode[]
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)